src/windows/MediaProxy.js

```diff
@@ -203,20 +234,64 @@ module.exports = {
         var id = args[0];
         var thisM = Media.get(id);
 
-        var normalizedSrc = thisM.src.replace(/\//g, '\\');
+        var success = function () {
+            Media.onStatus(id, Media.MEDIA_STATE, Media.MEDIA_STOPPED);
+            win();
+        };
+
+        var error = function (reason) {
+            Media.onStatus(id, Media.MEDIA_ERROR, reason);
+            lose(reason);
+        };
+
+        var fsTypes = {
+            PERSISTENT: 'PERSISTENT',
+            TEMPORARY: 'TEMPORARY'
+        };
+        var fsType;
+
+        var normalizedSrc = thisM.src;
+        normalizedSrc = setTemporaryFsByDefault(normalizedSrc);
+
+        if (normalizedSrc.indexOf('ms-appdata:///local/') === 0) {
```

Comment:
Do you mean cdvfile://localhost/persistent paths?
There is another [PR for this](https://github.com/apache/cordova-plugin-media/pull/57).